scripts/pkg/update-docs.go

```diff
@@ -179,13 +179,34 @@ func fixupMarkdownFile(path string) error {
 	if DryRun {
 		return nil
 	}
+
+	// Prefix link to be added before relative links in
+	// order to translate relative links into absolute links.
+	prefix := fmt.Sprintf("(https://github.com/antrea-io/antrea/blob/%s/", version)
+
+	// Translate relative links to directories build, pkg, hack, ci, CHANGELOG and LICENSE file into absolute links.
+	relativeLinksExp := regexp.MustCompile(`\([\./]*(build/|pkg/|hack/|ci/|CHANGELOG|LICENSE)+[A-Za-z0-9\.\-_/]*\)`)
+	relativeLinks := relativeLinksExp.FindAll(md, -1)
+	for _, rl := range relativeLinks {
+		var start int
+		existingLink := string(rl)
+		for i, char := range existingLink {
+			if char != '(' && char != '.' && char != '/' {
+				start = i
+				break
+			}
+		}
```

Comment:
I think you should use a capturing group in your regex to avoid having to do this

Reply:
done, PTAL